utilityscripts/tiffs/tiffconvert.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import subprocess
+import shutil
+
+import argparse
+
+# -------------------------------------------------------------------
+# config
+
+debug_output_level = 3 # max 5 (only errors show)
+
+# -------------------------------------------------------------------
+def msg(message, debug_level=3):
```

Comment:
Have a look at `logging` in the std library, allows essentially the same thing, just with an API like `logger.warn()`. http://docs.python.org/library/logging.html